src/bridge.py

```diff
@@ -1,184 +1,87 @@
 #!/usr/bin/env python3
 
 import rospy
-from std_msgs.msg import Float64
-from sensor_msgs.msg import Imu
+import numpy as np
+import math
+
+from auv_msgs.msg import ThrusterForces, DeadReckonReport
 from geometry_msgs.msg import Wrench, PoseArray, Pose, Vector3, Quaternion 
 from sbg_driver.msg import SbgImuData, SbgEkfQuat
-import numpy as np
+from sensor_msgs.msg import Imu
+from std_msgs.msg import Float64
 from tf import transformations
-import math
 
 
 DEG_PER_RAD = 180/np.pi
 ANGLE_CHANGE_TOL = 90 
 euler = np.array([0.0, 0.0, 0.0])
 
-d = 0.2222 #m
```

Comment:
If we use the same values from state_aggregator, then we need to update clarke's size in the sim. 0.2222 and 0.5588 were approximations because we were simplifying clarke's shape to a cylinder. Should be easy to fix. (I think the thrusters are already in the correct position, but it's probably better to double check)